packages/video_player/pubspec.yaml

```
@@ -0,0 +1,28 @@
+name: video_player_tizen
+description: Tizen implementation of the video_player plugin
+homepage: https://github.com/flutter-tizen/plugins
+version: 1.0.0
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        pluginClass: VideoPlayerTizenPlugin
+        fileName: video_player_tizen_plugin.h
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  test: ^1.3.0
+  flutter_test:
+    sdk: flutter
+  integration_test: ^0.9.2
+  integration_test_tizen:
+    path: ../integration_test
+  pedantic: ^1.8.0
+
+environment:
+  sdk: ">=2.8.0<3.0.0"
+  flutter: ">=1.12.13+hotfix.5 <2.0.0"
```

This needs update to be published on pub.dev.

```suggestion
  flutter: ">=1.20.0 <2.0.0"
```

---

I will modify it.